certification/internal/policy/container/less_than_max_layers.go

```diff
@@ -0,0 +1,54 @@
+package container
+
+import (
+	"fmt"
+
+	cranev1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	acceptableLayerMax = 40
+)
+
+// UnderLayerMaxCheck ensures that the image has less layers in its assembly than a predefined maximum.
+type UnderLayerMaxCheck struct{}
```

Comment:
I think the file name and the check name are confusing for this check, and this one doesn't seem to follow the conventions of the other one. However of all the check, the rest have more natural names by the nature of what the check is doing.